MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/labs/SpinningSymbolLayerIconActivity.java

```diff
@@ -70,12 +70,13 @@ public void onMapReady(final MapboxMap mapboxMap) {
         getResources().getDrawable(R.drawable.hurricane_icon)))
 
       // Adding a GeoJson source for the SymbolLayer icons.
-      .withSource(new GeoJsonSource(SOURCE_ID, loadGeoJsonFromAsset("spinning_icon.geojson")))
+      .withSource(new GeoJsonSource(SOURCE_ID, "asset://spinning_icon.geojson"))
```

Comment:
with this setup it will try to load `"asset://spinning_icon.geojson"` as a geojson, you will need to wrap it with a URI

Reply:
oh, right. Thanks.